tutorials/3d/procedural_geometry/introduction.rst

```diff
@@ -0,0 +1,106 @@
+.. _doc_procedural_geometry_introduction:
+
+Introduction to procedural geometry
+===================================
+
+There are many ways to procedurally generate geometry in Godot. In this tutorial series
+we will explore a few of them. Each technique has its own benefits and drawbacks, so 
+it is best to understand each one and how it can be useful in a given situation. 
+
+What is geometry?
+-----------------
+
+Geometry is a fancy way of saying shape. In computer graphics, geometry is typically represented
+by an array of positions called "vertices". In Godot, geometry is represented by Meshes.
```

Comment:
If the target audience is beginner level it might be worth spelling out that:

 * A *position* refers to a point/location/position in 2D or 3D space. (Where 2D might be screen space and 2D/3D may be global/local "virtual" space.)  (e.g. (x,y) or (x,y,z) or (1,3) or (4, 2, 7) )
 * A point is also known as a "vertex" (with the plural "vertices" or "vertexes").

The non-obvious English language mapping from "vertex" to "vertices" I think contributes to making this topic more confusing for beginners than it actually needs to be. (I suspect this is also significant for people for whom English is not their first language.) (In addition to the fact I suspect many people encounter the concept of a "point" before knowing it as a "vertex".)

(Another point of potential confusion is how the word/concept of "Vector" does/doesn't fit into the terminology.)

Anyway, hope this feedback is helpful. It seems like a good idea to get feedback on WIP text like this while also keeping in mind that writing the complete 1st draft is the initial most important part before polishing. :) So my intention is to hopefully provide useful feedback in terms of audience-level/structure while also generally encouraging you to continue. :)

Reply:
While these are good considerations. I dont believe this is the correct place to get into the weeds on those subtle differences. By the time a user reaches this tutorial they will undoubtedly have had some experience and can be expected to know what a position is. I think it is enough to define "vertices" as an array of positions. 

Keep in mind the purpose if this tutorial series is to teach the user how to use arraymeshes, immediategeometry, surfacetool and meshdatatool. Not to serve as a comprehensive introduction to computer graphics.